ui/src/dashboards/actions/index.ts

```diff
@@ -628,25 +628,21 @@ export const hydrateTempVarValuesAsync = (
     const dashboard = getState().dashboardUI.dashboards.find(
       d => d.id === dashboardID
     )
-
-    const tempsWithQueries = dashboard.templates.filter(
-      ({query}) => !!query.influxql
-    )
-
-    const asyncQueries = tempsWithQueries.map(({query}) =>
-      getTempVarValuesBySourceQuery(source, {
-        query: makeQueryForTemplate(query),
+    const templates: Template[] = dashboard.templates
+    const queries = templates
+      .filter(template => !!template.query.influxql)
```

Comment:
How do you feel about using a get or getter here? Could query ever not exist on a template?

Reply:
Oo nice catch, totally.